pkg/rdt/rdt_test.go

```diff
@@ -431,6 +1280,12 @@ func TestBitMap(t *testing.T) {
 			t.Errorf("expected err but got %#x when converting %q", b, s)
 		}
 	}
+
+	// Test MarshalJSON
+	if s, err := Bitmask(10).MarshalJSON(); err != nil {
+	} else if string(s) != `"0xa"` {
```

Comment:
Shouldn't this bail out also in the theoretical case of JSON marhsalling failures ?

Reply:
Yes, of course. Clearly a mistake. Fixed.